src/components/Connect/ConnectedSuccessBox.tsx

```diff
@@ -1,3 +1,5 @@
+import { Connection } from '@generated/api/src';
```

Comment:
nit: if we import our types through the same place like below `src/services/api` will be less code to change in the future.